app/routes/_auth+/join.tsx

```diff
@@ -100,7 +100,9 @@ export async function action({ request }: ActionFunctionArgs) {
     signUpResult.status === "Email verification_required"
   ) {
     // Redirect to the email verification page using Remix's redirect function
-    return redirect(`/verify-email?email=${encodeURIComponent(email)}`);
+    return redirect(
+      `/otp?email=${encodeURIComponent(email)}&mode=confirm_signup`
```

Comment:
Here its `confirm_signup` but below its `signup`. Is that expected?

Reply:
`confirm_signup` is when the user do a normal signup using email and password.